USERGUIDE.md

```diff
@@ -870,6 +870,21 @@ func handleDeepLink(components: URLComponents) -> Bool
 }
 ```
 
+### How do I clear the session data for Mini Apps?
+
+In the case that a user logs out of your App, you should clear the session data for all of your Mini Apps. This will ensure that the next user does not have access to the stored sensitive information about the previous user such as Local Storage, IndexedDB, and Web SQL.
+
+The session data can be cleared by using the following:
+
+```swift
+WKWebsiteDataStore.default().removeData(ofTypes: WKWebsiteDataStore.allWebsiteDataTypes(), 
```

Comment:
This is fine for now I guess. But we should have a separate ticket to remove/clear the session only for the Mini Apps from WKWebsiteDataStore